packages/node_modules/@ciscospark/widget-recents/src/components/spaces-list/index.js

```diff
@@ -0,0 +1,24 @@
+import React, {PropTypes} from 'react';
```

Comment:
Probably could do that with quite a few of our public ones too, especially all of the list stuff for message